e2e_tests/integration/params.spec.ts

```diff
@@ -121,4 +119,17 @@ function runTests() {
     cy.resultContains('point({srid:4326, x:12.9082, y:57.7346})')
   }
   // })
+
+  // :params
+  // it('can set :params with multiple lines syntax', () => {
```

Comment:
Is this meant to be commented out ?

Reply:
yes, for now, this is because the file was rewritten when wanted to run all test cases with and without web worker. Probably the file can be rewritten so we still can keep the 'it' so we can get better test output. But I will not do that work in this PR.